### DISCUSSION TITLE
init terminating in do_boot

### DISCUSSION THREAD

**Discussion by Chetan-K-Desai** [apache/couchdb#4478] [General]
Typing to configure couchdb cluster and setup connectivity between 2 nodes
 ### **{"init terminating in do_boot",{'cannot get bootfile','D:\\Program Files\\Apache CouchDB/bin/start.boot'}}**

i got above error by running this command as per below from couchdb doc guide line
**### erl -name bus@192.168.29.120 -setcookie 'brumbrum' -kernel inet_dist_listen_min 9100 -kernel inet_dist_listen_max 9200** 

Please help me or guide me to setup connectivity between 2 nodes

* CouchDB version used: 3.3.1
* Operating system and version: windows 10 pro

**Comment by big-r81** [comment]
You installed CouchDB via the msi installer? Can you install CouchDB into a path without spaces an test again?

**Comment by Chetan-K-Desai** [comment]
Yes i install it by msi, and without any space,

I already try that,
Any more guidance! Please

**Comment by big-r81** [comment]
OK, let us do the following:

1. Completly uninstall CouchDB
2. Install CouchDB into a directory without spaces
3. Start CouchDB (sigle node), login into Fauxton (web interface) and run run "Verify CouchDB Installation"
4. Paste the result

If this is all working, please show, what do you change to configure a cluster (changed config files, etc.).

**Comment by Chetan-K-Desai** [comment]
Can you please tell me how can I setup multi node cluster in couchdb,

Like from beginning like create 2 vms or install in each vm!

If you tell me this step by step it will very helpful for me because I'm unable to understand couchdb docs

**Comment by big-r81** [comment]
Hi,

I moved this topic to discussions.

Please don't let us mix things here. You did not answer my last questions, so I don't know, if CouchDB starts normally on Windows for you. Before we setting up a cluster, we need to make sure, your normal setup is fine...
